src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+class SearchBar extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      input: ''
+    }
+  }
+
+  handleChange = (event) => {
+    this.setState({[event.target.name]: event.target.value})
+  }
+
+  render() {
+    return (
+      <form>
```

Comment:
Nice work with this! I'm looking forward to pulling all these components together.